### ISSUE TITLE
Conditional import

### ISSUE THREAD

**Issue by samselikoff** [mike-north/ember-lodash#16]
I'm trying to figure out if it's possible to use this library in my addon, but optionally *not* import lodash into the host app's final `vendor.js`. I use this in Ember CLI Mirage, which is primarily enabled in development/testing. 

Is there a strategy I can take for a Mirage option be used at build-time to exclude lodash? I'm happy to PR once I have a better idea. My broccoli-fu is weak.

**Comment by tchak** [comment]
I fully agree this is a huge problem. I suspect this is something faced by other addons.

I would like to try to solve it by looking in to `babel-plugin-lodash`. If I can find a way to include in builds only imported modules it would be a huge win.

**Comment by tchak** [comment]
I am just realising that using `babel-plugin-lodash` is not even remotely an option before `ember-cli` is updated to use babel 6...

**Comment by tchak** [comment]
I am so :sob: :sob: :sob: now

**Comment by samselikoff** [comment]
Going through old issues, is this a possibility now?

**Comment by tchak** [comment]
Even with babel 6 `babel-plugin-lodash` may not be trivial to use with `ember-cli`. My best bet would be to follow the `ember-cli` tree shaking effort when it's ready.

**Comment by lolmaus** [comment]
Hi! I'm thinking about an addon with test helpers that would rely on lodash. If the host app does not depend on lodash directly, then my addon should not include lodash in `production` environment.

I can imagine `ember-lodash` importing lodash conditionally by checking an option in host app's `ember-cli-build.js`. This should be trivial to implement?
